### ISSUE TITLE
Test is failing with all django versions

### ISSUE THREAD

**Issue by awais786** [django-es/django-elasticsearch-dsl#377]
I am running these tests locally and getting this failure.

```
ERROR: test_custom_generate_id_is_called (tests.test_documents.DocTypeTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/awaisqureshi/Documents/devstack/django-elasticsearch-dsl/.tox/py27-django-111-es74/lib/python2.7/site-packages/mock/mock.py", line 1330, in patched
    return func(*args, **keywargs)
  File "/Users/awaisqureshi/Documents/devstack/django-elasticsearch-dsl/tests/test_documents.py", line 94, in test_custom_generate_id_is_called
    data = json.loads(mock_bulk.call_args[0][0].split("\n")[0])
IndexError: tuple index out of range
```

**Comment by natabene** [comment]
@safwanrahman Could you please have a look at this at your earliest convenience. I believe it is blocking this https://github.com/django-es/django-elasticsearch-dsl/pull/371 . Thanks so much in advance!

**Comment by safwanrahman** [comment]
This should be fixed by 0cc33bbcfcf4f931b1dae537a320eaa2bf672307
